src/components/Popover/Popover.jsx

```diff
@@ -12,24 +12,29 @@ export const Popover = React.forwardRef((props, ref) => {
   const {
     placement,
     children,
+    popoverHelperId,
     portalId,
     ...restProps
   } = props;
 
   const PopoverEl = (
-    <div
-      {...transferProps(restProps)}
-      className={classNames(
-        styles.root,
-        ref && styles.isRootControlled,
-        getRootSideClassName(placement, styles),
-        getRootAlignmentClassName(placement, styles),
-      )}
-      ref={ref}
-    >
-      {children}
-      <span className={styles.arrow} />
-    </div>
+    <>
+      {!!popoverHelperId && <div className={styles.helper} id={popoverHelperId} popover="auto" />}
```

Comment:
I think we should add a comment explaining why we use this hack and how it works.

Reply:
Please reformat the code like this:

```jsx
{!!popoverHelperId && (
  <div
    className={styles.helper}
    id={popoverHelperId}
    popover="auto" 
  />
)}
```